src/Alert.php

```diff
@@ -40,72 +44,65 @@ final class Alert extends Widget
     private string $iconText = '';
     private string $layoutHeader = '';
     private string $layoutBody = '{body}{button}';
-    private array $parts = [];
 
     /**
-     * The HTML attributes for the main widget tag.
-     *
-     * @param array $value Array of attribute name => attribute value pairs.
+     * Returns a new instance with the HTML attributes. The following special options are recognized.
      *
-     * @return static
+     * @param array $values Attribute values indexed by attribute names.
      */
-    public function attributes(array $value): self
+    public function attributes(array $values): self
```

Comment:
```suggestion
    public function attributes(array $valuesMap): self
```

Reply:
Optional and subjective. But for me personally it helps to quickly differentiate a list from a map just by looking at the variable's name.